recipes/hdf5/all/conanfile.py

```diff
@@ -87,6 +90,11 @@ def requirements(self):
         if self.options.parallel:
             self.requires("openmpi/4.1.0")
 
+    def build_requirements(self):
+        if Version(self.version) >= "1.14.0":
+            self.tool_requires("cmake/3.25.0")
```

Comment:
HDF5 1.14.0 is actually happy with just CMake 3.18.
Should I rewrite the above as:

```python3
    def build_requirements(self):
        if Version(self.version) >= "1.14.0":
            self.tool_requires("cmake/[>3.18.0]")
```